apps/gruenbuch/templates/gruenbuch_overview_page.html

```diff
@@ -19,8 +19,34 @@ <h1>{{ page.page_title }}</h1>
     {{ self.page_intro|richtext }}
 </section>
 <div id="layout-grid__area--maincontent">
-  {% for block in self.body %}
-      {{ block }}
-  {% endfor %}
+    {% for block in self.body %}
+        {{ block }}
+    {% endfor %}
+    {% if settings.apps_settings.Downloads.gruenbuch %}
+        {% with gruenbuch=settings.apps_settings.Downloads.gruenbuch %}
```

Comment:
Can we call this variable download? The gruenbuch confused me. :woman_shrugging: